### PR TITLE
Zs 212 fix microgrids tests

### PR DESCRIPTION
**Pull request by smvdheijden**

### CODE REVIEW

**Comment by smvdheijden** [comment]
retest this please

**Comment by smvdheijden** [comment]
retest this please

File: cucumber-tests-platform/src/test/resources/cucumber-platform.properties [hunk 1]
```diff
@@ -66,10 +66,10 @@ response.delay.random.range=2000
 
 #IEC61850
 #ICD file from osgp-protocol-simulator-iec61850 resources
-iec61850.mock.icd.filename=Pampus_v0.4.5_reporting_hack.icd
-iec61850.mock.icd.filename.marker.wadden=MarkerWadden_0_1_1.icd
-
-iec61850.mock.port=60102
+#iec61850.mock.icd.filename=Pampus_v0.4.5_reporting_hack.icd
+#iec61850.mock.icd.filename.marker.wadden=MarkerWadden_0_1_1.icd
+#
```
Comment (riannefijn) [inline cucumber-tests-platform/src/test/resources/cucumber-platform.properties]: Delete please

File: cucumber-tests-platform/src/test/resources/features/osgp-adapter-ws-microgrids/GetDataAlternativeScenarios.feature [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+      | NumberOfSystems          |        1 |
+      | SystemId_1               |        1 |
+      | SystemType_1             | PV       |
+      | NumberOfMeasurements_1   |        1 |
+      | MeasurementId_1_1        |        1 |
+      | MeasurementNode_1_1      | Health   |
+      | MeasurementQualifier_1_1 |     1024 |
+      | MeasurementValue_1_1     |      3.0 |
+
+  @TestThis @Iec61850MockServerPampus
```
Comment (riannefijn) [inline cucumber-tests-platform/src/test/resources/features/osgp-adapter-ws-microgrids/GetDataAlternativeScenarios.feature]: Remove @TestThis please

File: cucumber-tests-platform/src/test/resources/features/osgp-adapter-ws-microgrids/GetDataAlternativeScenarios.feature [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+      | SystemId_1                |          1 |
+      | SystemType_1              | BOILER     |
+      | NumberOfMeasurements_1    |          1 |
+      | MeasurementFilterNode_1_1 | Health     |
+    Then a SOAP fault should be returned
+      | DeviceIdentification | RTU-PAMPUS        |
+      | Component            | PROTOCOL_IEC61850 |
+      | Message              | ???               |
+
+  @TestThis @Iec61850MockServerPampus
```
Comment (riannefijn) [inline cucumber-tests-platform/src/test/resources/features/osgp-adapter-ws-microgrids/GetDataAlternativeScenarios.feature]: Remove @TestThis please

**Pull request merged**